src/main/java/propertyFiles/MimeTypeFacets.properties

```diff
@@ -170,6 +170,7 @@ application/x-7z-compressed=Archive
 application/x-xz=Archive
 application/warc=Archive
 application/x-iso9660-image=Archive
+application/vnd.eln+zip=ELN Archive
```

Comment:
I have some doubt about this. We try to put file types in a few buckets...

![Screen Shot 2023-02-09 at 8 53 44 AM](https://user-images.githubusercontent.com/21006/217831821-13189952-3724-4132-9661-58726a75ac22.png)

... so I'm not sure we should create a new type, "ELN Archive", or not. That is, maybe it should be lumped under "Archive".

Another approach would be to put it under "Data".

I guess I'm ok with "ELN Archive" (though not everyone will know what ELN stands for) but I'm curious if anyone feels strongly about this.

Reply:
I've pushed a commit in that direction, I agree that there is no need to create another bucket.